receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -414,7 +416,18 @@ func (jr *jReceiver) startCollector(host receiver.Host) error {
 	}()
 
 	// And finally, the gRPC server
-	jr.grpc = grpc.NewServer()
+	if jr.config.CollectorGRPCTLSSettings != nil {
+		option, err := jr.config.CollectorGRPCTLSSettings.ToGrpcServerOption()
```

Comment:
This call to `ToGrpcServerOptions` is in the start path. Can we move it to creattracereceiver or newtracereceiver method?

Reply:
yep!